### PR TITLE
Ellipsis

### PR DESCRIPTION
**Pull request by KamilWasilonek**
https://kamilwasilonek.github.io/ui-internship-lwo2/html/Ellipsis/

### CODE REVIEW

File: html/Ellipsis/index.html [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+          Throughout your project, our designers create and implement visual designs and workflows, solicit user feedback, and work with you to make sure what gets built is what is needed. 
+        </p>
+      </div>
+    </section>
+  </main>
+  
+  <!-- Cards -->
+  <section class="cards">
+    <button type="button" class="card card-1">
+      <p>
```
Comment (tsvetkovv) [inline html/Ellipsis/index.html]: `p` cannot be inside `button`

File: html/Ellipsis/index.html [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+          <br>
+          Throughout your project, our designers create and implement visual designs and workflows, solicit user feedback, and work with you to make sure what gets built is what is needed. 
+        </p>
+      </div>
+    </section>
+  </main>
+  
+  <!-- Cards -->
+  <section class="cards">
+    <button type="button" class="card card-1">
```
Comment (tsvetkovv) [inline html/Ellipsis/index.html]: I think it's not button, just `a` element. They don't look as button

File: html/Ellipsis/index.html [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+        <!-- Title -->
+        <h4 class="action__title">
+          development
+        </h4>
+  
+        <!-- Description -->
+        <p class="action__description">
+          Our design practice offers a full range of services including branding strategy,
+          interaction and visual design and user experience testing. 
+          <br>
```
Comment (tsvetkovv) [inline html/Ellipsis/index.html]: please avoid using br except poems and addresses
https://stackoverflow.com/a/1726103
use `<p>` for splitting by paragraphs

File: html/Ellipsis/index.html [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+        </p>
+      </div>
+
+      <!-- Development -->
+      <div class="what-we-do__action">
+        <!-- Action image -->
+        <picture class="services__icon">
+          <source media="(min-width: 1200px)" srcset="./assets/development/shape-3@3x.png">
+          <source media="(min-width: 768px)" srcset="./assets/development/shape-3@2x.png">
+          <img src="./assets/development/shape-3.png" alt="Development">
```
Comment (tsvetkovv) [inline html/Ellipsis/index.html]: I think it's just decorative icon, `alt` attribute should be empty
https://support.siteimprove.com/hc/en-gb/articles/115000013031-Accessibility-Image-Alt-text-best-practices

File: html/Ellipsis/style.css [hunk 5]
```diff
@@ -0,0 +1,490 @@
...
+
+/* ------------------------------------------------ About us ------------------------------------------------ */
+.about-us {
+  position: relative;
+  padding: 70px 10%;
+  text-align: center;
+}
+
+.about-us__title {
+  position: absolute;
```
Comment (tsvetkovv) [inline html/Ellipsis/style.css]: For every section (about us, services, what to do etc) you reinvent title styling — not scalable solution. Please think how to make it universal

File: html/Ellipsis/index.html (hunk 3 above)
Comment (tsvetkovv) [reply html/Ellipsis/index.html]: and please take care of every entry of `br`

**Review by tsvetkovv — CHANGES_REQUESTED** [review]

File: html/Ellipsis/index.html [hunk 6]
```diff
@@ -0,0 +1,249 @@
...
+    </button>
+  
+  </section>
+
+  <main>
+
+    <!-- About us -->
+    <section class="about-us">
+      <!-- Header -->
+      <header class="about-us__title section-title">
```
Comment (tkachukivan) [inline html/Ellipsis/index.html]: it should be h* tag

File: html/Ellipsis/index.html [hunk 7]
```diff
@@ -0,0 +1,249 @@
...
+        about us
+      </header>
+
+      <!-- Description -->
+      <div class="about-us__description">
+        <p class="description__text">
```
Comment (tkachukivan) [inline html/Ellipsis/index.html:60]: there is bigger distance between lines in mockup for all text blocks

File: html/Ellipsis/index.html [hunk 8]
```diff
@@ -0,0 +1,249 @@
...
+
+    <!-- Services -->
+    <section class="services">
+      <!-- Header -->
+      <header class="services__title section-title">
+        services
+      </header>
+
+      <!-- Boder top line -->
+      <div class="services__line"></div>
```
Comment (tkachukivan) [inline html/Ellipsis/index.html:82]: maybe possible to do as for about section without additional element

File: html/Ellipsis/style.css [hunk 9]
```diff
@@ -0,0 +1,492 @@
...
+}
+
+.banner__mouse {
+  width: 20px;
+}
+
+@media screen and (min-width: 768px) {
+  .banner {
+    background: linear-gradient(rgba(0, 0, 0, 0.2), rgba(0, 0, 0, 0.2)),
+    url("./assets/banner/layer-8@2x-min.png") no-repeat right center;
```
Comment (tkachukivan) [inline html/Ellipsis/style.css]: it is good to have some space to not being confused with properties

File: html/Ellipsis/index.html [hunk 10]
```diff
@@ -0,0 +1,249 @@
...
+          </p>
+          <p>
+            The outcome of this scoping session is a proposal or high-level statement of work. 
+            Based on your requirements, we can provide a complete offering or just the pieces you need. 
+          </p>
+      </div>
+    </section>
+
+    <!-- What we do -->
+    <section class="what-we-do">
```
Comment (tkachukivan) [inline html/Ellipsis/index.html]: try not to name classes by content, as content can change

**Review by tsvetkovv — APPROVED** [review]

**Review by tkachukivan — APPROVED** [review]

**Pull request merged**